eulxml/xmlmap/cerp.py

```diff
@@ -30,7 +30,7 @@
 
 # CERP is described at http://siarchives.si.edu/cerp/ . XML spec available at
 # http://www.records.ncdcr.gov/emailpreservation/mail-account/mail-account_docs.html
-
+# schema is empty as of April 2016
```

Comment:
I guess this note is to indicate that the schema doesn't resolve? (or does it actually resolve, but is empty?)  Thanks for documenting that; the wording could maybe be a bit clearer.

I did a quick search, and there a schema at http://www.history.ncdcr.gov/SHRAB/ar/emailpreservation/mail-account/mail-account.xsd that looks like it is the right thing.  Should we go ahead and include it in the catalog? (with comments)  I think, in this case, maybe we should go ahead and add a copy to the repo in case that one disappears too.